rotkehlchen/chain/ethereum/decoding/pickle/decoder.py

```diff
@@ -0,0 +1,95 @@
+from typing import List, Optional
+
+from rotkehlchen.accounting.structures import (
+    HistoryBaseEntry,
+    HistoryEventSubType,
+    HistoryEventType,
+)
+from rotkehlchen.assets.asset import EthereumToken
+from rotkehlchen.chain.ethereum.decoding.pickle.constants import PICKLE_CONTRACTS
+from rotkehlchen.chain.ethereum.decoding.structures import ActionItem
+from rotkehlchen.chain.ethereum.structures import EthereumTxReceiptLog
+from rotkehlchen.chain.ethereum.utils import asset_normalized_value
+from rotkehlchen.constants.ethereum import ZERO_ADDRESS
+from rotkehlchen.types import EthereumTransaction
+from rotkehlchen.utils.misc import hex_or_bytes_to_address, hex_or_bytes_to_int
+
+
+def maybe_enrich_pickle_transfers(
+    token: Optional[EthereumToken],  # pylint: disable=unused-argument
+    tx_log: EthereumTxReceiptLog,
+    transaction: EthereumTransaction,
+    event: HistoryBaseEntry,
+    action_items: List[ActionItem],  # pylint: disable=unused-argument
+) -> bool:
+    """Enrich tranfer transactions to address for jar deposits and withdrawals"""
+    if not (
+        hex_or_bytes_to_address(tx_log.topics[2]) in PICKLE_CONTRACTS or
+        hex_or_bytes_to_address(tx_log.topics[1]) in PICKLE_CONTRACTS or
+        tx_log.address in PICKLE_CONTRACTS
+    ):
+        return False
+
+    # Deposit
+    if (
```

Comment:
Use the space you got. Don't add new lines for a comment if you got space. Same at the other places below

```suggestion
    
    if (  # Deposit
```